src/libs/antares/study/include/antares/study/area/store-timeseries-numbers.h

```diff
@@ -26,11 +26,16 @@
 
 namespace Antares::Data
 {
-template<int TimeSeriesT>
+template<TimeSeriesType TimeSeriesT>
```

Comment:
This is good, calling the generic version (through other template parameters than the ones specified) should lead to a linker error, thus preventing such calls.